src/Http/WebUtilities/test/QueryHelpersTests.cs

```diff
@@ -105,6 +105,14 @@ public void ParseEmptyOrNullQueryWorks(string? queryString)
             Assert.Empty(collection);
         }
 
+        [Fact]
+        public void ParseQueryRetainsLeadingAndTrailingWhitespace()
+        {
+            var collection = QueryHelpers.ParseQuery("? key = value &");
```

Comment:
This isn't a valid query so I don't think we need a test for it. Retaining escaped whitespace would be a more legitimate test.

Reply:
Makes sense. Removing this test. There are other tests about escaped whitespace.